packages/webapp/src/containers/Animals/Inventory/AnimalsBetaSpotlight.jsx

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { TourProviderWrapper } from '../../../components/TourProviderWrapper/TourProviderWrapper';
+import { setSpotlightToShown } from '../../Map/saga';
+import { showedSpotlightSelector } from '../../showedSpotlightSlice';
+import { Trans } from 'react-i18next';
+import Badge from '../../../components/Badge';
+import { ReactComponent as SendIcon } from '../../../assets/images/send-icon.svg';
+import styles from './styles.module.scss';
+
+export default function AnimalsBetaSpotlight({ children, setFeedbackSurveyOpen }) {
```

Comment:
Since we'll likely be needing similar "beta" spotlights for upcoming features, I could see value in having a `BetaSpotlight` reusable component with only the more specific parameters as props, and then having that component be a child of this one. Not a blocker for merge though, can be done when we have an upcoming beta feature!